tank_lab_to_nwb/convert_towers_task/towersprocessednwbconverter.py

```diff
@@ -44,3 +45,7 @@ def get_metadata(self):
             print(f"Warning: no subject file detected for session {session_id}!")
 
         return metadata
+
+    def get_spikeglx_metadata(self, spikeglx_filepath):
+        temp_spikeglx_interface = SpikeGLXRecordingInterface(**dict(file_path=spikeglx_filepath))
+        return temp_spikeglx_interface.get_metadata()
```

Comment:
Can you put this in the `get_metadata` method?

Reply:
With this design it requires the direct spikeglx_filepath, which is not used anywhere else in the converter so it can't pull it from there.

This functionality may or may not be resolved using .write_recording in the pipeline, I'll have to check how it interacts.

I might as well make this PR include that as well, just to see. Changing to draft in the meantime.

Reply:
Fixed